src/components/steps/step_horizontal.tsx

```diff
@@ -106,6 +107,7 @@ export const EuiStepHorizontal: FunctionComponent<EuiStepHorizontalProps> = ({
 
   return (
     <button
+      aria-disabled={buttonAriaDisabled}
```

Comment:
```suggestion
      aria-disabled={buttonAriaDisabled || undefined}
```

This will help with the snapshots / remove the attribute entirely if it isn't true

Reply:
also tbh I don't see the point of point of pulling this out to a const vs just inlining it, if we're not using it more than once
```suggestion
      aria-disabled={status === 'disabled' || undefined}
```

Reply:
That's a solid point about inlining for one instance. I'll update it and push with the changelog.